### ISSUE TITLE
Can we add a "legend" to the map?

### ISSUE THREAD

**Issue by evanwolf** [openoakland/soft-story-alameda#15]
A pane hovering over some waterway with provenance info. 
"Data collected by... XXX, between YYY and ZZZ"
"Soft Story defined as ‘AAA’ by Agency BBB"

**Comment by daguar** [comment]
:+1: Let's do it.

**Comment by jnmarcus** [comment]
@daguar is this related to the 'legend' we were talking about last night or different?

**Comment by daguar** [comment]
Yeah, I think this would just be showing the data sources and maybe a key showing what the different color markers mean.

**Comment by jnmarcus** [comment]
ok, and when you say the 'data sources', do you mean where the data was provided from? (like who/what organization gave it to us...)

**Comment by daguar** [comment]
Yup.

**Comment by jnmarcus** [comment]
:thumbsup: do you want to include this for v1?

**Comment by daguar** [comment]
Yeah, shouldn't be hard to do both.

**Comment by jnmarcus** [comment]
kool, has Andrea been added to the repo? She's not coming up when I try to tag her..

**Comment by amitch23** [comment]
Hey all, just joined, but I'm not listed as a contributor yet. I'll get started on the legend after I get branching under my belt.

**Comment by jnmarcus** [comment]
@amitch23 you'll be listed as a contributor once you've made changes and pushed those changes to the remote repository.  Were you able to clone the repo?

**Comment by amitch23** [comment]
Ok, got it. I cloned the repo and am working in a branch. Once I've finished the legend, I'll merge to the master - is that what you're doing @jnmarcus (i.e. merging your branches)?

**Comment by daguar** [comment]
Hey @amitch23! Welcome! The best approach would be to push your branch up to GitHub (ie, `git push origin my-branch-name`) and then create a Pull Request in the GitHub interface. Then someone else can review and merge.

**Comment by amitch23** [comment]
Ok, thanks @daguar for clarifying. 2 questions:
1. What should the actual text be for the data source and soft story building definition?
2. Are these the 4 categories? Has anyone started working on this and decided on colors?
"Potential Soft Story Building
Not Soft Story Building
Soft Story Building
Retrofitted & Rehabilitated"

**Comment by jnmarcus** [comment]
@amitch23 I think that's the working draft version for the categories.  Maybe try to touch base tonight with @dmieler to confirm.  

I don't think colors have been decided yet.  Maybe run it by @dmieler tonight as well, to see if she has any preference, other than that, feel free to choose them yourself.

**Comment by jnmarcus** [comment]
about the data-source and soft story building definition stuff, maybe for now just add placeholder text, until we have that stuff defined? @dmieler @daguar what are your thoughts?

**Comment by jnmarcus** [comment]
On second thought, I'm thinking also that that info may not really belong in the 'legend', and that it belongs in the pop-up cards for each individual building...

What do you guys think?

**Comment by amitch23** [comment]
@jnmarcus , I think it depends if there's going to be another area with static information/introductory/explanatory text. Where are you planning to put the search box? 

I created a pull request to merge the legend so far, if anyone wants to take a look. :)

**Comment by mmieler** [comment]
I agree with @jnmarcus about putting data sources (e.g., "Data collected by XXX, between YYY and ZZZ 'Soft Story' defined as AAA by Agency BBB") in the pop-up cards for each building. As we discussed last night, the pop-up card would also contain a more detailed description of the building category (i.e., if a building is labeled as soft story, the pop-up card would explain what a soft story is).

**Comment by evanwolf** [comment]
You might think about the legend doing a few jobs: 
- Common information you don't want to duplicate on individual cards. 
- Overall context. (So you don't have to figure it out by inspection)
- Links to static info (FAQ/About/Contact)
- Action links for the whole map (feedback/share/bookmark/discuss)

By including it within the map, your users don't even need to scroll down to see this info.

**Comment by daguar** [comment]
@amitch23 did a nice first pass on implementing this, and so what we currently have is:

![screen shot 2015-01-27 at 12 33 43 am](https://cloud.githubusercontent.com/assets/994938/5915584/2c76a678-a5bc-11e4-8654-9d1511393d38.png)

**Comment by jnmarcus** [comment]
nice! looks great!

**Comment by dmieler** [comment]
Looks good to me too!

Danielle
